e2e/e2e_tests.py

```diff
@@ -0,0 +1,151 @@
+import os
+from time import sleep
+import unittest
+import subprocess
+
+try:
+    import requests
+    from bs4 import BeautifulSoup
+except ImportError:
+    requirements_file = os.path.join(os.path.dirname(__file__), 'requirements.txt')
+    subprocess.check_call("pip install -r {}".format(requirements_file).split(' '))
```

Comment:
Forget what I wrote, it's not creating a virtualenv here anyway, so it doesn't make sense at all. Will update and use a message to the user what to do to get it up and running.